proto/quicksilver/airdrop/v1/messages.proto

```diff
@@ -24,12 +25,23 @@ message MsgClaim {
   string chain_id = 1 [ (gogoproto.moretags) = "yaml:\"chain_id\"" ];
   int32 action = 2 [ (gogoproto.moretags) = "yaml:\"action\"" ];
   string address = 3 [ (gogoproto.moretags) = "yaml:\"address\"" ];
-  bytes proof = 4
-      [ (gogoproto.moretags) = "yaml:\"proof\"", (gogoproto.nullable) = true ];
+  Proof proof = 4
```

Comment:
This is nicer than three distinct slices for proof / data / key - no risk of sorting errors.

But we do need to support the submission of multiple proofs in a single msg - as we want to sum the qOsmo (or whatever) across multiple pools.

Reply:
updated in last commit (+rebase) so should merge with latest PRs